backend/entityservice/tasks/encoding_uploading.py

```diff
@@ -43,12 +43,12 @@ def handle_raw_upload(project_id, dp_id, receipt_token, parent_span=None):
     def filter_generator():
         log.debug("Deserializing json filters")
```

Comment:
This log message looks  like a copy/paste error of mine no doubt. Could you update.